src/db/connection.py

```diff
@@ -1,3 +1,27 @@
+import psycopg2
+import psycopg2.pool
+import os
+pool = None
```

Comment:
Put this property and the following methods into a class and initialize the pool in the constructor

Reply:
I prefer leaving it module scoped because we should be able to instantiate multiple pools. The point of the pool is to provide a per-process set of connections that the app can use. Creating this as a class doesn't make sense, considering we shouldn't reinitialize the pool per instance

Reply:
Okay that makes sense, let's keep it then.